prime-router/src/test/kotlin/config/validation/JsonSchemaValidationServiceTest.kt

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.prime.router.config.validation
+
+import assertk.assertThat
+import assertk.assertions.isEmpty
+import java.io.File
+import kotlin.test.Test
+
+class JsonSchemaValidationServiceTest {
+
+    private val service = JsonSchemaValidationService()
+
+    @Test
+    fun organizations() {
+        val errors = service.validateYAMLFileStructure(
+            ConfigurationType.ORGANIZATIONS,
+            File("settings/organizations.yml")
```

Comment:
Now any changes to organizations.yml will trigger a test failure in CI (as well as any changes that break the schema or service).

Reply:
This test is not really testing JsonSchemaValidationServiceTest, but rather actually validating our settings. I think this is fine, but let's also create a test that tests `JsonSchemaValidationServiceTest`? I'm picturing hardcoded input that always evaluates to either true or false (two tests).

I would also add a comment to your existing test that explains it is expected it will fail if our settings files are not valid.